dev/Controller/SPController.php

```diff
@@ -42,28 +40,15 @@
  */
 final class SPController extends AbstractController
 {
-    private $identityProvider;
-    private $serviceProvider;
-    private $postBinding;
-
     public function __construct(
-        ServiceProvider $serviceProvider,
-        IdentityProvider $identityProvider,
-        PostBinding $postBinding
+        private readonly ServiceProvider $serviceProvider,
```

Comment:
Volgens mij kan de class dan readonly ipv alle props

Reply:
Given we extend the abstract controller, this is not an option. Maybe we should stop using that abstract controller.. Also one for the backlog. But given this is a dev resource. I'm not super concerned at this point.